packages/scene-composer/src/components/three-fiber/DataOverlayComponent/DataOverlayComponent.tsx

```diff
@@ -14,11 +16,20 @@ export interface DataOverlayComponentProps {
 
 export const DataOverlayComponent = ({ node, component }: DataOverlayComponentProps): ReactElement => {
   const sceneComposerId = useContext(sceneComposerIdContext);
+  const getObject3DBySceneNodeRef = useStore(sceneComposerId)((state) => state.getObject3DBySceneNodeRef);
+  const object3D = getObject3DBySceneNodeRef(node.ref);
+  const objectWorldPos = object3D?.getWorldPosition(new Vector3());
+  const objectSize = object3D && getSafeBoundingBox(object3D).getSize(new Vector3());
+  // TODO: fix for when tag auto rescale is on
```

Comment:
Non-blocker: Ideally, link an issue at this line.

Reply:
same here